src/components/Contact.tsx

```diff
@@ -0,0 +1,160 @@
+            vertical
+            style={{ padding: "2em", backgroundColor: "#E14E54" }}>
+            <Header as='h1'
+                    content='CONTACT US'
+                    style={{ fontWeight: "bold", marginBottom: "1em", textAlign: "center" }}/>
+            <Grid
+                divided
+                inverted
+                stackable
+                className="segment centered"
+                style={{ backgroundColor: "#E14E54" }}>
+                <Grid.Row>
+                    <Message
+                        success
+                        hidden={!success}
+                        compact
+                        header="Thanks for reaching out! We'll be in touch soon :)"
+                    />
+                    <Message
+                        error
+                        hidden={!error}
+                        compact
+                        header="Oh no! Looks like something went wrong."
+                        content="Sorry for the inconvenience. Please get in touch with hello@codefordenver.org directly."
+                    />
+                    <Message
+                        error
+                        hidden={!invalid}
+                        compact
+                        header="Oh no! Looks like an invalid email was provided."
+                        content="Please check email and try again."
+                    />
+                </Grid.Row>
+                <Grid.Row>
+                    <Form loading={loading}
+                          onSubmit={handleFormSubmit}>
+                        <Form.Field>
+                            <label
+                                className="white"
+                                style={label__css}>Name*</label>
+                            <input
+                                style={borderRadius__css}
+                                placeholder='Name'
+                                required
+                                value={name}
+                                onChange={(e) => setName(e.target.value)}/>
+                        </Form.Field>
+                        <Form.Field>
+                            <label
+                                className="white"
+                                style={label__css}>Email*</label>
+                            <input
+                                style={borderRadius__css}
+                                placeholder='Email'
+                                value={email}
+                                type='email'
+                                required
+                                onChange={(e) => setEmail(e.target.value)}/>
+                        </Form.Field>
+                        <Form.Field>
+                            <label
+                                className="white"
+                                style={label__css}>Message*</label>
+                            <TextArea
+                                style={borderRadius__css}
```

Comment:
Maybe look into smaller border radius, or set textarea to not resizable `resize: none`. Currently the resize triangle is outside of the textbox for the message field.